docs/types/value-types.rst

```diff
@@ -65,10 +65,10 @@ This means that, for example ``~int256(0) == int256(-1)``.
 Shifts
 ^^^^^^
 
-The result of a shift operation has the type of the left operand, truncating the result to match the type.
+The result of a shift operation has the type of the left operand, truncating the result to match the type. This means an "overflow" will not be "checked".
 Right operand must be unsigned type. Trying to shift by signed type will produce a compilation error.
 
-- For positive and negative ``x`` values, ``x << y`` is equivalent to ``x * 2**y``.
+- For positive and negative ``x`` values, ``x << y`` is equivalent to ``x * 2**y`` run in unchecked mode.
```

Comment:
This is indeed a good spot, this definitely has to be changed and/or clarified. Strictly speaking, this applies to the other cases as well, though, the exponentiation can just as well overflow in the right shift case and things still won't revert, whereas in checked arithmetic the supposedly "equivalent" expression would.